### PR TITLE
DOCSP-40874-amazon-platform

### PR DESCRIPTION
**Pull request by jason-price-mongodb**
JIRA:

https://jira.mongodb.org/browse/DOCSP-40874

Not possible to build. No web hook, and no rules to manually build. Asked in platform channel, and answer is: "docs-shared doesn't build on its own". Therefore staging isn't possible and cannot check how changes render.

Text update is per request from Alex Ettouati on Slack, with minor adjustments from me. Stuck with Alex's requested text very closely.

### CODE REVIEW

File: server/platform-support/platform-support.rst [hunk 1]
```diff
@@ -741,3 +741,12 @@
 
 .. [#oracle-support] On Oracle Linux, MongoDB only supports the Red Hat
    Compatible Kernel.
+
```
Comment (sarah-olson-mongodb) [inline server/platform-support/platform-support.rst:744]: This entire paragraph belongs in the Release Notes.

File: server/platform-support/platform-support.rst [hunk 2]
```diff
@@ -741,3 +741,12 @@
...
+Starting in MongoDB 8.0, new MongoDB Server versions (major and minor)
+support the minimum operating system (OS) minor version defined by the
+OS vendor. After an OS minor version is no longer supported by the OS
+vendor, MongoDB updates the MongoDB Server to support the next OS minor
+version. For details, see:
+
```
Comment (sarah-olson-mongodb) [inline server/platform-support/platform-support.rst:750]: (Content)
There is nothing about this paragraph that communicates a change in version from Amazon2023 to 2023.3 (x86 and arm64). What is your reasoning for omitting that?

**Review by sarah-olson-mongodb — COMMENTED** [review]
Thanks @jason-price-mongodb.
Couple questions for you.

File: server/platform-support/platform-support.rst (hunk 1 above)
Comment (jason-price-mongodb) [reply server/platform-support/platform-support.rst:744]: Per review comment, will only include in release notes.

File: server/platform-support/platform-support.rst (hunk 2 above)
Comment (jason-price-mongodb) [reply server/platform-support/platform-support.rst:750]: Per our discussion with PM and dev, will not doc minor versions.

**Comment by jason-price-mongodb** [comment]
Deleting this PR per Sarah only wanting the content in the server docs release notes.

**Comment by jason-price-mongodb** [comment]
@sarah-olson-mongodb please re-review. Alex requested in Slack it be added. I'll also ask him to confirm. I cannot add him as a reviewer, so will ask on Slack. Cheers!

**Review by AEttouati — APPROVED** [review]
LGTM

**Comment by jason-price-mongodb** [comment]
Per @AEttouati , will close without merge. On Slack, he states:
"yeah good to delete imo"

We also have this same content added to the 8.0 release notes:
https://www.mongodb.com/docs/upcoming/release-notes/8.0/#platform-support-updates

**Pull request closed without merging**